packages/minimongo/common.js

```diff
@@ -2,6 +2,13 @@ import LocalCollection from './local_collection.js';
 
 export const hasOwn = Object.prototype.hasOwnProperty;
 
+export class QueryError extends Error {
+  constructor(message) {
+    super(message);
+    this.name = 'QueryError';
```

Comment:
You could move it to the class definition instead of the construction:

```js
class QueryError extends Error {
  name = 'QueryError';
  constructor(message) {
//...
```